base_multi_image/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
```

Comment:
Well, as this has been totally fragmented, I doubt git blame give the proper information. But the facts are these:

* I started from product_multi_image from Sharoon Thomas.
* In one moment in the development, there were 0 lines of code from the original (and I'm not talking about API rewrite).
* Then, you took this and split it in a base + additional modules.

So for me it's clear: the copyright should be shared between you and me.